Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.8.6-alpine3.12
+
+ARG UID=1000
+ARG VERSION
+
+RUN set -eux ; \
+  adduser -D -u ${UID} -h /dropbox dropbox ; \
+  apk add --no-cache --virtual .build-deps \
+    gcc \
+    musl-dev \
+    python3-dev \
+    libffi-dev \
+    openssl-dev ; \
+  pip install maestral==${VERSION} ; \
+  pip cache purge ; \
+  apk del --no-network .build-deps
+
+USER dropbox
+VOLUME ["/dropbox"]
+WORKDIR /dropbox
+
+CMD ["maestral", "start", "-f"] 
```

Comment:
No, but we can pass env variables if that's help. I need to gain more experience with Maestral to think about the right solution :)

In my head, if I have e.g. a work-related Dropbox and a personal one, I'd probably want to run 2 separate Maestral in 2 separate container, which mount specific directories.

Reply:
If someone wants to run this in a single container, then I need a bit of time to experiment with that, but don't know why would someone do it instead of running two isolated containers, with probably different permissions.

Reply:
Fair point, running isolated containers probably makes more sense. If there are valid requests by users, we can still reconsider this.